feat(onesync/natives): add GetVehicleHornType (#2697)
citizenfx/fivem: ext/native-decls/GetVehicleHornType.md

---
ns: CFX
apiset: server
---
## GET_VEHICLE_HORN_TYPE

```c
Hash GET_VEHICLE_HORN_TYPE(Vehicle vehicle);
```

> This native is a getter for the client-side native `START_VEHICLE_HORN`, which allows you to return the horn type of the vehicle.

Reviewer: ```suggestion
This is a getter for the client-side native [`START_VEHICLE_HORN`](https://docs.fivem.net/natives/?_0x9C8C6504B5B63D2C), which allows you to return the horn type of the vehicle.
```

**Note**: This native only gets the hash value set with `START_VEHICLE_HORN`. If a wrong hash is passed into `START_VEHICLE_HORN`, it will return this wrong hash.

```c
enum eHornTypes
{
    NORMAL = 1330140148,
    HELDDOWN = -2087385909,
    AGGRESSIVE = -92810745

> }

Reviewer: ````suggestion
}
```
````
Missing closing brackets

## Parameters
* **vehicle**: The vehicle to check the horn type.

## Return value

> The vehicle horn type hash. Returns 0 if the vehicle does not have a horn type set.

Reviewer: ```suggestion
Returns the vehicle horn type hash, or `0` if one is not set.
```
